tests/types/npm-test.json

```diff
@@ -254,6 +254,316 @@
       "expected_output": "pkg:npm/core@2.0.1#googleapis/api/annotations",
       "expected_failure": false,
       "expected_failure_reason": null
+    },
+    {
+      "description": "Negative Test: An npm purl with an un-encoded @ in scope should fail parsing",
+      "test_group": "advanced",
+      "test_type": "parse",
+      "input": "pkg:npm/@angular/animation@12.3.1",
+      "expected_output": null,
+      "expected_failure": true,
```

Comment:
Is this true? Last I heard it was expected to parse because there was no instruction to specifically reject unencoded `@` characters in positions where they _could_ require encoding depending on the rest of the PURL.